packages/hdwallet-keepkey-tcp/src/transport.ts

```diff
@@ -12,6 +12,7 @@ export class TransportDelegate implements keepkey.TransportDelegate {
     this.config = {
       ...config,
       headers: {
+        'Authorization': config?.headers?.serviceKey || '',
```

Comment:
This will already be included if you set `config.headers.Authorization`... not sure there's any value in including it separately here. If we do, though, we should add it to the `Config` type explicitly.